### PR TITLE
Test on latest PHP versions

### PR DESCRIPTION
**Pull request by roback**
These are the versions that are supported currently according to: https://www.php.net/supported-versions.php

I haven't tested 8.0 locally, so I have no idea whether it works or not 🙈

### CODE REVIEW

**Comment by roback** [comment]
> I haven't tested 8.0 locally, so I have no idea whether it works or not 🙈

As suspected, it doesn't 😅

**Comment by walro** [comment]
> As suspected, it doesn't 😅

The testing framework is too old IIRC :)

**Comment by roback** [comment]
> The testing framework is too old IIRC :)

Probably. I knew I shouldn't have touched this repo 😃

**Comment by roback** [comment]
At least it works in 7.3 and 7.4 now. Only 8.0 to go 😄

**Comment by roback** [comment]
> At least it works in 7.3 and 7.4 now. Only 8.0 to go 😄

There we go 🎉

File: tests/ClientTest.php [hunk 1]
```diff
@@ -67,7 +66,7 @@ public function testExecuteQueryWithInvalidAPIKey() {
             $c->execute_query($q);
             $this->fail('Should throw AuthException.');
         } catch (AuthException $e) {
-            $this->assertContains('Unauthorized', $e->getMessage());
+            $this->assertStringContainsString('Unauthorized', $e->getMessage());
```
Comment (walro) [inline tests/ClientTest.php:69]: this method name rolls right of the tongue 😆

**Review by Pontus4 — APPROVED** [review]
To me it looks good!

**Pull request merged**